docs/source/configuration/standard-attributes.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: OpenTelemetry standard attributes
+subtitle: Attach standard attributes to router telemetry
+description: Attach OpenTelemetry standard attributes to Apollo Router telemetry
+---
+
+import RouterServices from '../../shared/router-lifecycle-services.mdx';
+
+[OpenTelemetry semantic conventions](https://opentelemetry.io/docs/specs/semconv/) define a set of **standard attributes** that can be attached to [spans](./spans), [instruments](./instruments) and [events](./events). These attributes are used to filter and group data in your application performance monitor (APM).
+
+The attributes available depend on the service of the router pipeline.
+
+For example, set the standard attribute `http.response.status_code` on the router span:
+
+```yaml title="router.yaml"
+telemetry:
+  spans:
+    router:
+      attributes:
+        # Standard attributes
+        http.response.status_code: true
+```
+
+### Configuration reference 
+
+<RouterServices />
+
+Each service supports a unique set of standard attributes.
+
+#### Router
+
+Standard attributes of the `router` service:
+
+| Attribute                      | Value                               | Description                                                                                                                                        |
+|--------------------------------|-------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|
+| `error.type`                   |                                     | Describes a class of error the operation ended with.                                                                                               |
```

Comment:
I'd remove the periods from the descriptions that have them since none of these are complete sentences.